### PR TITLE
Remaining part1

### PR DESCRIPTION
**Pull request by itsmebipul00**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,4 @@
-REACT_APP_JWT_SECRET= bipul123456
\ No newline at end of file
+REACT_APP_JWT_SECRET= bipul123456
+
+REACT_APP_CLIENT_ID= AS9LIT3RuJawe-TLqR5p-lD9GCJ_EApZRA_-MHlnebuVpf3djXWyT73YQj_3AyBCcLmcCY-bFm5pXPDB
+
```
Comment (Ankur9669) [inline .env]: fix: You should remove the env file

File: src/Components/NewAddressDialog/NewAddressDialog.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    backdrop-filter: blur(2px) brightness(30%);
+    z-index: 9999;
+    inset: 0% 0% 0% 0%;
+}
+
+.submit-btn{
+    padding: 0.35rem 1rem ;
+}
+
+@media (min-width: 40rem){
```
Comment (Ankur9669) [inline src/Components/NewAddressDialog/NewAddressDialog.css:29]: fix: use @media only screen and max-width syntax

File: src/Components/Search/Search.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import { LogoProvider } from '../../assets/Icons/Icons'
+
+import { useProducts } from '../../actionProviders/productActions'
+
+import { useState } from 'react'
+
+import { Link } from 'react-router-dom'
+
+import { debounce } from '../../utils/debounce'
+
```
Comment (Ankur9669) [inline src/Components/Search/Search.js:14]: fix: Use barrel exports

File: src/Components/Search/Search.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	const searchFilters = debounce(text => {
+		setSearch(text)
+	}, 1000)
+
+	const searchedProducts = products?.filter(pro =>
+		pro.title.toLowerCase().includes(search)
+	)
+
+	console.log(searchedProducts)
```
Comment (Ankur9669) [inline src/Components/Search/Search.js]: fix: remove console.log statement

**Review by Ankur9669 — APPROVED** [review]
Coding conventions and the UI looks good to me, You can look to the suggestions after that you can merge

File: src/Components/NewAddressDialog/NewAddressDialog.css (hunk 2 above)
Comment (itsmebipul00) [reply src/Components/NewAddressDialog/NewAddressDialog.css:29]: it is mobile first

File: src/Components/Search/Search.js (hunk 4 above)
Comment (itsmebipul00) [reply src/Components/Search/Search.js]: fixed

File: .env (hunk 1 above)
Comment (itsmebipul00) [reply .env]: fixed

**Pull request merged**